### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by Lukazo0**

### CODE REVIEW

**Comment by Lukazo0** [comment]
@nrios14 you're welcome 😊 I edited another part of the doc directly to current PR.

**Review by michal-at-travisci — APPROVED** [review]
I did review and amended necessary sections

**Review by michal-at-travisci — DISMISSED** [review]

**Review by michal-at-travisci — APPROVED** [review]

File: user/reference/overview.md [hunk 1]
```diff
@@ -17,15 +17,46 @@ Each build runs in one of the following virtual environments.
...
-* [Ubuntu Bionic 18.04](/user/reference/bionic/)
-* [Ubuntu Xenial 16.04](/user/reference/xenial/) **default**
-* [Ubuntu Trusty 14.04](/user/reference/trusty/)
-* [Ubuntu Precise 12.04](/user/reference/precise/)
+#### Full VM
 
-LXD compliant OS images for arm64 are run in [Packet](https://www.packet.com/). Have a look at [Building on Multiple CPU Architectures](/user/multi-cpu-architectures) for more information.
+This is sudo enabled, full virtual machine per build, that runs Linux
+
+* Spins up slower (increased build time compared to LXD container) yet without any [limitations](/user/reference/overview/#linux-security-and-lxd-container).
```
Comment (nrios14) [inline user/reference/overview.md:26]: Slow spin-up

File: user/reference/overview.md [hunk 2]
```diff
@@ -17,15 +17,46 @@ Each build runs in one of the following virtual environments.
...
+#### Which one do I use?
+
+For the majority of cases, whenever available, we recommend to use LXD-based containers. 
+
+Use full VM only if LXD is not available or you need 
+* privileged fs access
+* specific system call interception
+* hugepages support (subject to changes on short notice)
+
+Below table sums up available Ubuntu environments and virtualization type per CPU architecture:
```
Comment (nrios14) [inline user/reference/overview.md:45]: **The** table below sums up **the** available Ubuntu environments

File: user/reference/overview.md [hunk 3]
```diff
@@ -17,15 +17,46 @@ Each build runs in one of the following virtual environments.
...
+| [Ubuntu Bionic 18.04](/user/reference/bionic/) | `arch: amd64`: full VM only, default option<br />`arch: arm64`: LXD only<br />`arch: arm64-graviton2`: LXD only<br/>`arch: ppc64le`: LXD only<br/>`arch: s390x`: LXD only  |
+| [Ubuntu Xenial 16.04](/user/reference/xenial/) **default** | `arch: amd64`: full VM only, default option<br />`arch: arm64`: LXD only<br />`arch: arm64-graviton2`: LXD only<br/>`arch: ppc64le`: LXD only<br/>`arch: s390x`: LXD only  |
+| [Ubuntu Trusty 14.04](/user/reference/trusty/) | `arch: amd64`: full VM only, default option  |
+| [Ubuntu Precise 12.04](/user/reference/precise/) | `arch: amd64`: full VM only, default option  |
+
+
+LXD compliant OS images for arm64 are run on [AWS](https://aws.amazon.com/) and in [Packet](https://www.packet.com/). LXD compliant OS images for IBM Power and Z are run in [IBM Cloud](https://www.ibm.com/cloud). Have a look at [Building on Multiple CPU Architectures](/user/multi-cpu-architectures) for more information.
```
Comment (nrios14) [inline user/reference/overview.md:56]: **For more information see** Building on ...

**Review by nrios14 — COMMENTED** [review]
Did some minor English changes, I could not edit directly this time, so I commented the suggestions

File: user/reference/overview.md (hunk 1 above)
Comment (michal-at-travisci) [reply user/reference/overview.md:26]: Updating directly on master, thanks!

File: user/reference/overview.md (hunk 3 above)
Comment (michal-at-travisci) [reply user/reference/overview.md:56]: updating on master, thanks!

**Comment by michal-at-travisci** [comment]
Much appreciated @nrios14 - I was force pushing and I am adding fixes in the master now

**Pull request merged**